config/initializers/constants.rb

```diff
@@ -0,0 +1,4 @@
+LEVEL1 = 'beginner'
+LEVEL2 = 'intermediate'
+LEVEL3 = 'advanced'
+POSSIBLE_LEVELS = [LEVEL1, LEVEL2, LEVEL3]

```

Comment:
In my opinion its better to write code like this 
POSSIBLE_LEVEL = ['beginner, intermediate, advanced] - in this file
and use 
POSSIBLE_LEVEL [1] -for example in helper and view.